src/util/source_location.cpp

```diff
@@ -93,3 +94,25 @@ std::ostream &operator << (
   out << source_location.as_string();
   return out;
 }
+
+/*******************************************************************\
+
+Function: is_build_in
```

Comment:
There's another one here: https://github.com/diffblue/cbmc/blob/master/src/goto-instrument/cover.cpp#L1087
I'm sure there might be yet a few more...